www/script/project/helpers.js

```diff
@@ -540,6 +539,8 @@ define(function (require) {
                 sortable.push([objs[item], css_classes[orderItem][2], item]);
             }
             return sortable.sort(function(a, b) {
+                if(a[1] == b[1])
```

Comment:
Could you do equality without type conversion, I mean `===`.
The difference between this both is the following:
* equality symbol `==` compare a value of objects, so if you compare `0 == ""` you'll get `true`, so javascript do an implicit conversion of objects.
* equality symbol without conversion `===` compare type and value of objects, it means that `0 === ""` is not true